<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>1821</width>
      <height>941</height>
     </rect>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>260</y>
       <width>491</width>
       <height>611</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="val_Wallet">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="val_Current_Price">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="val_1USD">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="val_Contracts_Max">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>260</y>
       <width>341</width>
       <height>611</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="lable_Wallet">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Wallet</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Current_Price">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Current_Price</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_1USD">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1USD</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Contracts_Max">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Contracts_Max</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>260</y>
       <width>381</width>
       <height>611</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="rightMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="label_Contracts_Ratio">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Contracts_Ratio</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Contracts_Open">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Contracts_Open</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Unrealised_PNL">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Unrealised_PNL</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Entry_Price">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Entry_Price</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>1280</x>
       <y>260</y>
       <width>461</width>
       <height>611</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="val_Contracts_Ratio">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="val_Contracts_Open">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="val_Unrealised_PNL">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="val_Entry_Price">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>821</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_Leverage">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Leverage</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Leverage">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_Leverage">
        <property name="font">
         <font>
          <family>맑은 고딕 Semilight</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>입력</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_Cancel">
        <property name="font">
         <font>
          <family>맑은 고딕 Semilight</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>중지</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>47</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
